src/app/components/mocks/products.mock.ts

```diff
@@ -1,14 +1,14 @@
-import { Product } from '../../modules/shared/types/product';
+import { OrderProduct } from '../../modules/shared/types/product.order';
```

Comment:
Do you still need this mock data? You should work by now only with the data provided by the server. If this data is not used I would suggest to delete the file.